cmake/projects/Boost/schemes/url_sha1_boost.cmake.in

```diff
@@ -59,18 +59,23 @@ foreach(x ${boost_libs})
   list(APPEND boost_list "--without-${x}")
 endforeach()
 
-if("@MSVC@")
-  set(env_cmd "@HUNTER_MSVC_VCVARSALL@" "@HUNTER_MSVC_ARCH@")
-else()
-  set(env_cmd "")
-endif()
-
 if(CMAKE_HOST_WIN32)
   set(install_cmd "b2")
-  set(bootstrap_cmd "bootstrap.bat")
+  if("@MSVC@")
+    file(WRITE "@HUNTER_PACKAGE_DOWNLOAD_DIR@/patched_boostrap.bat" "\
```

Comment:
My attemp # 3 : Please move this code into file so we can reuse it in both `url_sha1_boost_library.cmake.in` and `url_sha1_boost.cmake.in`. Also this will help with readability since there will be no need to use escape symbols and explicit end-of-line symbols.

Reply:
I did the modification
it's testing : 
https://travis-ci.org/Ubiquite/hunter/builds/313423398
https://ci.appveyor.com/project/Ubiquite/hunter/build/1.0.8

Reply:
Okay, it's just not clear to me should I start testing, should I start review...